mewarpx/mewarpx/testing_util.py

```diff
@@ -67,3 +70,120 @@ def change_to_warpxdir(wd):
     comm.Barrier()
 
     return origwd
+
+
+def get_min_wmargin(array, margin=0.1):
+    """Get a minimum value with a multiplicative margin of safety.
+
+    Arguments:
+        array (np.ndarray): Array to find minimum of
+        margin (float): Multiplicative safety margin
+
+    Returns:
+        minwmargin (float): If min < 0, this is min*(1-margin). If min > 0, this
+            is min*(1+margin).
```

Comment:
Sphinx syntax this shouldn't be indented (eg `is` is even with `minwmargin`). Multi-line arguments do get indented because it's needed to identify when the next argument starts, Returns doesn't (at least last I used it). *shrug*.

Reply:
This was copied directly from metools.util so we should keep in mind to fix it there then as well.